Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildProjectReference.cs

```diff
@@ -72,20 +79,20 @@ public string ProjectPath
                 }
 
                 // Determine the absolute path of the MSBuild project (based on the projectPath being relative to the MSBuildProjectReference asset).
-                if (!string.IsNullOrEmpty(this.projectPath))
+                if (!string.IsNullOrEmpty(projectPath))
```

Comment:
This was automatic formatting. I can add this backm if you like?

Reply:
Preferably yes. We don't have consistent styling in the code right now, but I would prefer to qualify in at least the code I contribute to because I find it much easier to quickly reason about.

Reply:
Definetly, sorry, it's my automatic rules. Will fix this file.